app/src/main/java/net/skyscanner/backpack/demo/compose/CardStory.kt

```diff
@@ -97,10 +93,7 @@ fun LargeCornersCardExample(
 }
 
 @Composable
-@Preview
```

Comment:
Removed Previews from these stories – we generate a lot of previews by just using ViewStory or ComposeStory